README.md

```diff
@@ -1 +1,49 @@
-# todo-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> This is a todo list project done using ES6 modules and webpack
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- webpack
+
+## Live Demo
+
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy this link `https://github.com/ericmahare/todo-list.git`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write `git clone https://github.com/ericmahare/todo-list.git`.
+- go to the repository folder in your command prompt `cd awesome-books-modules`
```

Comment:
- [ ] [OPTIONAL] Great job with the set of instructions on your README, however, you still need to tell the user to run `npm i` and then `npm start` in order to be able to run the project successfully on their machine.